src/effects/index.ts

```diff
@@ -64,6 +64,16 @@ type ConfettiOptions = {
      * use gradient instead of solid particle color
      */
     gradient: boolean,
+};
+type FireworksOptions = {
+    /**
+     * max fireworks count
+     */
+    maxCount: number,
+    /**
+     * gravity value that firework adds to shift from it's start position
+     */
+    gravity: number,
```

Comment:
for definitions of `type`s and `interface`s we use semicolons on the lines rather than commas - I'll fix this before merging.